curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/understand-the-constructor-property.md

```diff
@@ -30,8 +30,7 @@ function joinBirdFraternity(candidate) {
 }
 ```
 
-**Note**  
-Since the `constructor` property can be overwritten (which will be covered in the next two challenges) it’s generally better to use the `instanceof` method to check the type of an object.
+**Note:** Since the `constructor` property can be overwritten (which will be covered in the next two challenges) it’s generally better to use the `instanceof` method to check the type of an object.
```

Comment:
Not sure about wrapping "constructor" here.

Reply:
This refers to the actual `.constructor` property on the object, so IMO should be wrapped.